src/cljs/main/broadfcui/page/workspace/method_configs/launch_analysis.cljs

```diff
@@ -82,7 +82,7 @@
    [:div {:style {:marginTop "1em"}}
     [comps/Checkbox
     {:ref "callCache-check"
-     :label "Use Call Caching"
+     :label [:span {:style {:marginBottom "0.8em"}} "Use Call Caching. " [:a {:href (config/call-caching-guide-url) :target "_blank"} "Learn More"]]
```

Comment:
Are there any established conventions for how to display such help elements? And would it be possible to include a screenshot in the PR? Without that it's hard to visualize the result.